lib/generate.js

```diff
@@ -34,3 +35,24 @@ fs.writeFileSync(
     noClaims,
   })
 )
+
+try {
+  fs.writeFileSync(
+    `${__dirname}/../out/blog/index.html`,
+    Mustache.render(fs.readFileSync('templates/blog.html').toString(), {
+      feeds,
+    })
+  )
+} catch (err) {
+  if (err.code === 'ENOENT') {
+    fs.mkdir(`${__dirname}/../out/blog`)
```

Comment:
This is crappy.

You can modify the build script to clean the folder everytime and then recreate the folder here everytime.

Also this is async and it's possible that the script below will run first. Use sync version of this.